k8s/namespaces/rd/rd-professional-api-integration/rd-professional-api-integration.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/professional-api:prod-023945d-20210824144722
+      image: hmctspublic.azurecr.io/rd/professional-api:prod-023945d-20210824144722   #{"$imagepolicy": "flux-system:rd-professional-api-integration"}
```

Comment:
Can you change this policy to rd-location-ref-data-load? It looks to be using the same, I would remove the image repo/policy for rd-professional-api-integration

Reply:
Image policy/repo now updated.